clayui.com/content/blog/2019-10-24-introducing-clay-v3.md

```diff
@@ -0,0 +1,130 @@
+
+```shell
+MAJOR.MINOR.PATCH
+```
+
+So you can see more often releases of `patch` and consequently` major` but that does not mean there will be big changes like v3 introducing a new rewrite. We hope these releases will be more continuous and help us iterator faster.
+
+### Migrate Guide
+
+We have prepared a migration guide from v2 to v3, keep in mind that v3 has had major changes this guide helps you come from Metal.js to React.js. We hope this can ease the pain for those unfamiliar with React.js.
+
+<div class="clay-ullist clay-ullist-margin-sm">
+
+- [Migrate the Clay components from v2 to v3](/docs/migration/migrate-the-clay-components-from-v2-to-v3.html)
+
+</div>
+
+### 3.x Roadmap
+
+We are working on developer experience enhancements and delivering new features to accelerate development. We have a few things for our roadmap.
+
+#### Data Display
+
+To help teams create complex and extensible data displays, Clay offers different Table components that can be used and configured on a high level.
+
+We will collect use cases and possible future data display cases, we will work on creating documentation and examples to help developers effectively display data across different applications.
+
+#### Data Visualization
+
+The proposal here is to analyze the problems we had with ClayCharts, many teams complained about its use, API, inconsistencies with Lexicon… the idea is to listen to all the people who worked with Charts to gather feedback and from that we can make a decision how let's work on this, whether it's a charts lib, a lib from scratch, API-only enhancements…
+
+ClayCharts should be simple to create charts that Lexicon predicts and easy to customize to get out of specification.
+
+An important point is performance, many onscreen charts should not be a problem, We will study the cases and help solve the performance issues.
+
+The criteria to get there:
+
+<div class="clay-ullist clay-ullist-margin-sm">
+
+- Performance
+- Flexibility
+- Consistency with Lexicon
+- Readable API
+
+</div>
+
+We are already working on [collecting feedbacks and you can leave your 2cents](https://github.com/liferay/clay/issues/2577).
+
+#### Layout and ClayManager components
+
+Still under research, we will be working to understand common component use cases that can become components in Clay that help develop Layouts more easily.
+
+Since we have several contexts for different components it can be a bit annoying to add many nested contexts to your project's root application. We will work in an aggregating context that will bring all our contexts together in one place, making it easier for teams that use Clay as their base and not just some components.
+
+#### A11y component
+
+A written component on top of react-axe that gives direct accessibility feedback on screen and only in the development environment. Inspired by the work for the new Facebook.
```

Comment:
Maybe we should include a link to our storybook to show where people can see that integration.

Reply:
Hmm would be a good idea, but do you just say submit to the storybook or when we implement this new component?

Reply:
oh wait, i think I misread this. Are you saying we are evaluating a new A11y component down the road?

Reply:
yes I added this in in the roadmap doc

Reply:
ah gotcha. For that case then, I might actually leave it out since its fairly vague at the moment. If we want to include something on accessibility, we could just comment broadly how our goal is to continually increase accessibility of all of our components.

Reply:
Hm, it makes sense. We can post a post soon just talking about it.